### DISCUSSION TITLE
Trading/working days

### DISCUSSION THREAD

**Discussion by ncooder** [vacanza/holidays#1916] [Q&A]
Is there a straightforward way to calculate working days for different countries? I have noticed that for some countries, like Bangladesh, the weekend days are listed as Saturday and Sunday, when they should actually be Friday and Saturday. How can I ensure the correct weekend days are used for each country?

**Comment by arkid15r** [comment]
Could you split your question set into separate threads? I believe the Bangladesh one has already been addressed.
